.travis.yml

```diff
@@ -42,7 +42,7 @@ env:
 matrix:
   include:
     - compiler: clang
-      env: SCANBUILD="scan-build --status-bugs" CFLAGS=-D_REENTRANT CONFIG_EXTRA=--enable-code-coverage MAKE_TARGET=check
+      env: SCANBUILD="scan-build --status-bugs" CFLAGS=-D_REENTRANT CONFIG_EXTRA="--enable-code-coverage --enable-defaultflags" MAKE_TARGET=check
```

Comment:
This adds `--enable-defaultflags` to `CONFIG_EXTRA`. Based on the commit message I was expecting `--disable-defaultflags`. Am I missing something?

Reply:
Yeah, their's also an issue with the check for code cov here:
https://travis-ci.org/tpm2-software/tpm2-abrmd/jobs/578610194#L1771

I'll have to update that as well to be a sub-string check.

Reply:
face palm, that moment you realize you had it right locally and never updated the branch the PR is based off of.